tests/test_compare.py

```diff
@@ -20,6 +21,21 @@ def _config() -> Config:
 # ===== Unit Tests =========================================
 
 
+@pytest.mark.parametrize(
+    "value, expected, is_equal",
+    [(Unit("meter"), Unit("meter"), True), (Unit("meter"), Unit("feet"), False)],
```

Comment:
What about cases for "percent" and "fraction"?

Reply:
[was able to get rid of needing to spell out "percent" and "fraction"](https://github.com/MORSECorp/snappiershot/pull/96/commits/5e743c9bcbaebe11b2c77c1050831607853a458d)